Makefile

```diff
@@ -0,0 +1,18 @@
+clean_translations_temp_directory:
+	rm -rf I18N/I18N/
+
+
+pull_translations:
+	make clean_translations_temp_directory
+	atlas pull $(ATLAS_OPTIONS) translations/openedx-app-ios/I18N/I18N/:I18N/I18N/
```

Comment:
The `atlas pull` command represents a real command which will work once we implement the `combine_translations` step.